CHANGES/1015.feature.rst

```diff
@@ -0,0 +1 @@
+Implemented support for the free-threaded build of CPython 3.13 -- by :user:`lysnikolaou`.
```

Comment:
I think we need another change note of the `packaging` type saying
```rst
Started publishing wheels made for the free-threaded build of CPython 3.13 -- by :user:`lysnikolaou`.
```

@bdraco WDYT?

Reply:
LGTM